file_example.c

```diff
@@ -76,6 +77,28 @@ static bool file_check_config(const char *cfgstring, char **reason)
 	return true;
 }
 
+static int file_reconfig(struct tcmu_device *dev, uint32_t cfgtype)
+{
+	int ret = 0;
+
+	switch (cfgtype) {
+	case NO_RECONFIG:
+		tcmu_err("Nothing to reconfigure\n");
+		break;
+	case CONFIG_PATH:
+		tcmu_err("Device Path is an Unsupported Reconfiguration\n");
```

Comment:
You can drop the per handler error messages and just put it in reconfig_device. When yo do this, try to sync up your style sometimes you capitalize the first letter like above, but below for write cache you didn't.

Also use tcmu_dev_err, so we know what device its coming from.